### PR TITLE
[zh] Chinese patch-6

### LINKED ISSUE #1365: Some date and time fields do not use localized format.
Some date and time fields do not use localized format.

For example:

"join_time" field on the user profile page

image

The format of time used in this field is hard-coded to "MMMM YYYY", but in Chinese we use "YYYY年 MMMM".

Maybe we nead a new field in the locales files, for Moment.js do not support localized formats that only contain month and year.

More examples are coming.

### PR DESCRIPTION
**Pull request by kj415j45**
- [x] beatmappacks
- [x] minor updates
- [x] fix #1365

### CODE REVIEW

File: resources/lang/en/users.php [hunk 1]
```diff
@@ -53,6 +53,7 @@
         '404' => 'User not found! ;_;',
         'age' => ':age years old',
         'current_location' => 'Currently in :location',
+        'dateFormat' => 'MMMM YYYY',
```
Comment (kj415j45) [inline resources/lang/en/users.php]: i think put it here is ok ¯\\_(ツ)\_/¯

File: resources/lang/zh/beatmappacks.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    否则这些谱面可能无法正常游玩。',
+                'scary' => '不要',
+            ],
+            'note' => [
+                '_' => '强烈建议:scary，因为较老的谱面的质量可能比现在差一些。',
+                'scary' => '下载最新的谱面包',
+            ],
+        ],
+        'title' => '谱面包',
+        'description' => '以特定主题打包好的谱面包', //需要帮助
```
Comment (benpigchu) [inline resources/lang/zh/beatmappacks.php]: 【 “围绕某个相同主题打包好的谱面包” 】

File: resources/lang/en/users.php (hunk 1 above)
Comment (nekodex) [inline resources/lang/en/users.php]: Use camel case here, also name it more explicitly, e.g. `'joined_at_format' => ...`

Comment (notbakaneko) [inline resources/lang/en/users.php]: I think you mean snake_case, right? >_>

Comment (nekodex) [inline resources/lang/en/users.php]: Uh yes, whoops I pulled a nanaya. >.>

Comment (nanaya) [inline resources/lang/en/users.php]: I don't think it's user specific so it should be in common. The syntax is momentjs specific so I think it should say it somewhere in the key.

Comment (nekodex) [inline resources/lang/en/users.php]: It's specific to the display of that single field though, which is only used in the user profile (in both client and web)

Comment (nanaya) [inline resources/lang/en/users.php]: it's just another variation of date format? (year+month)

Comment (kj415j45) [inline resources/lang/en/users.php]: so should i move it to common?

and name it `common.time.date_format.M_Y` ?

**Comment by kj415j45** [comment]
waiting for #1369 and #1370

File: resources/lang/zh/beatmappacks.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'index' => [
+        'blurb' => [
+            'important' => '下载前必读',
+            'instruction' => [
+                '_' => '安装：下载好谱面包之后,直接解压 .rar 文件到你的 osu! 的 Songs 文件夹下。
```
Comment (Gennadiyev) [inline resources/lang/zh/beatmappacks.php]: 您 (?)

File: resources/lang/zh/beatmappacks.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                    所有的谱面此时都是 .zip 或 .osz 文件, osu! 会在你下一次启动时自动载入这些谱面，
```
Comment (Gennadiyev) [inline resources/lang/zh/beatmappacks.php]: 您 (?)

File: resources/lang/zh/beatmappacks.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+                    因此:scary自己解压这些谱面。
+                    否则这些谱面可能无法正常游玩。',
+                'scary' => '不要',
+            ],
+            'note' => [
+                '_' => '强烈建议:scary，因为较老的谱面的质量可能比现在差一些。',
+                'scary' => '下载最新的谱面包',
+            ],
+        ],
+        'title' => '谱面包',
```
Comment (Gennadiyev) [inline resources/lang/zh/beatmappacks.php]: 曲包 (?) Medals 这么用了

File: resources/lang/zh/beatmappacks.php [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        'description' => '围绕某个相同主题打包好的谱面包',
```
Comment (Gennadiyev) [inline resources/lang/zh/beatmappacks.php]: 同上 曲包？

File: resources/lang/zh/beatmappacks.php [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    ],
+
+    'show' => [
+        'download' => '下载',
+        'item' => [
+            'cleared' => '玩过',
+            'not_cleared' => '没玩过',
```
Comment (Gennadiyev) [inline resources/lang/zh/beatmappacks.php]: 建议 已完成 未完成

File: resources/lang/zh/layout.php [hunk 8]
```diff
@@ -52,7 +53,7 @@
             'show' => '信息',
             'index' => '列表',
             'artists' => '杰出艺术家',
-            // 'getPacks' => 'packs',
+            'packs' => '谱面包',
```
Comment (Gennadiyev) [inline resources/lang/zh/layout.php]: 曲包

File: resources/lang/zh/users.php [hunk 9]
```diff
@@ -146,10 +152,10 @@
                 ],
             ],
             'me' => [
-                'title' => '关于我!',
+                'title' => '关于我！',
             ],
             'medals' => [
-                'empty' => '该用户还没有获得奖牌. ;_;',
+                'empty' => '该用户还没有获得奖牌。;_;',
```
Comment (Gennadiyev) [inline resources/lang/zh/users.php]: 统一 奖牌 为 奖章。

File: resources/lang/zh/beatmappacks.php (hunk 4 above)
Comment (Gennadiyev) [inline resources/lang/zh/beatmappacks.php]: 曲包(另外这里也要改)

File: resources/lang/zh/beatmappacks.php (hunk 7 above)
Comment (kj415j45) [inline resources/lang/zh/beatmappacks.php]: 不好，否决

**Pull request merged**